### ISSUE TITLE
Update version of bcrypt and sqlite3 to 5.0.1 and 5.0.0 respectively

### ISSUE THREAD

**Issue by damiansan239** [w3tecch/express-typescript-boilerplate#237]
I want to use this opportunity to say thank you to the wonderful template you built here. I have been reading it as if it were a textbook. It has made a big influence on how design my project both in TS and to be frank, in my python flask code.

While I was trying to setup, the bcrypt and sqlite3 deps gave big issues. Am not the only one having this problem and it's not just in this project.
I ask that the version of the dependencies be moved to the version I mention as this seems to be a nice spot.
Thank you

**Comment by vanenshi** [comment]
look like they don't maintain this boilerplate anymore
I updated the packages myself, thinking about creating a fork of this wonderful boilerplate with upgraded packages.

also, some of the packages are deprecated and there is a replacement for them

**Comment by nunnally** [comment]
@vanenshi Can you share your upgraded version, please?

**Comment by vanenshi** [comment]
> @vanenshi Can you share your upgraded version, please?

yeah sure, I'm going to open a PR and add the most recent version of packages. ill notify you then

**Comment by SayGus** [comment]
i also updated myself. this is best boilerplate i've founded. 
@vanenshi waiting for your fork

**Comment by dweber019** [comment]
Thx. But actually we don't maintain it currently, as we use other technologies right now.

**Comment by YonatanCatav** [comment]
I'm really curious what kind of technologies

**Comment by sampathkatari** [comment]
@vanenshi were you able to fork and create this boiler plate with upgraded packages? If so can you please share it here?

**Comment by vanenshi** [comment]
> @vanenshi were you able to fork and create this boiler plate with upgraded packages? If so can you please share it here?

Actully i am not using this poilerplate anymore. I am using nestjs with a custom made boilerplate, inspired by NarHakobyan/awesome-nest-boilerplate
